index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
+    <h1 id="title">Survey Form</h1>
+    <p id="description">improve our site and forms</p>
+    <form id="survey-form">
+      <label id = "name-label"
+      for name = "name">Name</label><br>
+      <input type="text" id="name"
+      placeholder="enter your name" required><br><br>
+      <label id = "email-label"
+      for name = "email">Email</label><br>
+      <input type="email" id="email"
+      placeholder="enter your email" required><br><br>
+      <label for="number" id="number-label">Age</label>
```

Comment:
- [ ] Please use classes next to id tags to work easily with more categories with the same name, it also provides a lot of ease in identifying items for different sets of tags, unlike id tags that don't give their properties to more than one tag.